policy-controller/k8s/index/src/namespace.rs

```diff
@@ -0,0 +1,45 @@
+use crate::{authz::AuthzIndex, pod::PodIndex, server::SrvIndex, DefaultAllow};
+use std::collections::HashMap;
+
+#[derive(Debug)]
+pub(crate) struct NamespaceIndex {
+    pub index: HashMap<String, Namespace>,
+
+    // The global default-allow policy.
+    default_allow: DefaultAllow,
+}
+
+#[derive(Debug)]
+pub(crate) struct Namespace {
+    /// Holds the global default-allow policy, which may be overridden per-workload.
```

Comment:
nit/tioli: should this be
```suggestion
    /// Holds the global default-allow policy for this namespace, which may be 
    /// overridden per-workload.
```
or something? Since the `NamespaceIndex` struct has the _truly_ global default policy.